MIGRATION.md

```diff
@@ -2,6 +2,25 @@
 
 These guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
+## `3.0.1` -> `3.1.0`
```

Comment:
I think we usually leave open the next version, so `3.0.1` -> `next-version`

Reply:
But the CHANGELOG etc. references `3.1.0`? Why is MIGRATION different?

Reply:
hm, interesting, I hadn't notice the changelog had that.

Reply:
`3.1.0` is the next release which is going out today. We had to release multiple release candidates before 3.1.0, therefore in order to keep track of how many rc were released I added a not in the changelog referencing what the next release candidate was going to be.

Reply:
This now says `Next Version` correctly